design/pen.behavior.md

```diff
@@ -0,0 +1,79 @@
+# Pen Behavior
+
+The pen behavior is a method of recording the path by which a user moves a grabbed object that they activate so that various scenearios 
+can be carried out that wish to capture the virtual drawing of a user.  This behavior will carry with it client side visuals
+to ensure low-latency feedback, but will supply the complete curve data needed by the MRE app to acto on once the drawing is done.
+The `PenBehavior` will be an extension of the `ToolBehavior`.
+
+## Architecture
+
+There will be a new `PenBehavior` that extends the `ToolBehavior` that will have the ability to generate a wet ink effect along the use 
+path on the client and provide the use path to the MRE app for generating a spline curve asset that could have a mesh to applied to it
+before sending it to all the clients to render. and a mesh to apply to this spline curve.
+
+### Behavior
+``` ts
+/**
+ * Pen behavior class containing the target behavior actions.
+ */
+export class PenBehavior extends ToolBehavior {
```

Comment:
You're going to hate me for saying this, but I don't think we need a `PenBehavior`. `ToolBehavior` + transform streaming via Actor gives us everything we need to make a pen behavior. (transforms streaming from a child actor at an offset, if an offset is desired).

Reply:
I don't hate you, but don't totally agree with you either.  I feel I don't have a compelling enough case though to push it further.  I will remove, but will not forget...

Reply:
I actually am circling back to this and going to provide a valid use case for having the pen behavior as well as an API on the pen behavior.  We should discuss real time online to hash this out.

Reply:
I would like to know your use case here, too - I shared Eric's opinion about ToolBehavior+transformStreaming=PenBehavior

Reply:
Had a great call and the usefulness of the pen behavior is a bit more clear.  I have updated the design here for the recording API as discussed in the call, and am further updating based on the results of that meeting.